app/code/VConnect/OrderVolume/Model/Order/Config.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace VConnect\OrderVolume\Model\Order;
+
+class Config
+{
+    public const SALES_ORDER_TABLE = 'sales_order';
```

Comment:
I don't think you will need those constants if you rework your flow and will use another event for a setting volume to order and items.